File: Iniciante/1038 - Lanche/Python/1038.py
def main():
    price_table = {1: 4.00, 2: 4.50, 3: 5.00, 4: 2.00, 5: 1.50}

    item, amount = map(int, input().split())
    price = price_table[item]
    total = amount * price
    print(f"Total: R$ {total:.2f}")


if __name__ == "__main__":
    main()

File: Iniciante/1038 - Lanche/TypeScript/1038.ts
import * as fs from "fs";

function main(): void {
    const input: string = fs.readFileSync("/dev/stdin", "utf8");
    const [item, amount]: number[] = input.split(" ").map(Number);
    const menu = {
      1: 4.0,
      2: 4.5,
      3: 5.0,
      4: 2.0,
      5: 1.5,
    };
  const price: number = menu[item];
  const total: number = amount * price;
  console.log(`Total: R$ ${total.toFixed(2)}`);
}

main();
